### ISSUE TITLE
Feature Request: Tabs

### ISSUE THREAD

**Issue by samsebastian** [kaliber5/ember-bootstrap#41]
I've noticed that there are pretty much zero ember addons that provide an easy way to implement tabs, so I'm assuming it's a hard thing to replicate, but it would be awesome to get access to tabs available in this package which has so many other things from Bootstrap that I'm used to! :+1:

**Comment by discane** [comment]
:+1: to this request.

**Comment by simonihmig** [comment]
Actually I think it is not that hard to implement one, just found this example: https://github.com/lfridael/ember-tab-example

The problem with a generic approach is that there are different ways for different needs: one might use a different routes for every active tab, and have only the active tabs content be rendered into the DOM by the subroute template. Or the content of all tab panels is in the same template, without routing support...

However I will think about if there is a good way or a general approach that makes those different solutions possible.

**Comment by brianjgeiger** [comment]
+1 for at least the case of tabs without routing support.

**Comment by samsebastian** [comment]
I think the best way to mirror the Bootstrap framework would be to start with non-routed tabs. 
Some sort of interface like (ignore the naming, I'm just making it up without thought):

```
{{#bs-tabs-container}}
  {{#bs-nav-tabs}}
    {{bs-nav-tab for="tabOne" label="Tab One"}}
    {{bs-nav-tab for="tabTwo" label="Tab Two"}}
  {{/bs-nav-tabs}}

  {{#bs-tabs}}
    {{#bs-tab name="tabOne"}}
      This is tab one content.
    {{/bs-tab}}
    {{#bs-tab name="tabTwo"}}
       And the content for two here.
    {{/bs-tab}}
  {{/bs-tabs}}
{{/bs-tabs-container}}
```

I'm not experienced enough to create the actual code for the above, but that is what immediately came to mind when I thought of the request.

**Comment by ngelx** [comment]
+1 to this 👍

**Comment by simonihmig** [comment]
released in v0.8.0
